From 0b43e43b949d09c5275821345018c1b08f648d85 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.1-0b43e43b949-20210324T021049Z-avarab@gmail.com>
In-Reply-To: <20210307132905.14212-1-avarab@gmail.com>
References: <20210307132905.14212-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sat, 6 Mar 2021 23:53:30 +0100
Subject: [PATCH v2] mktag tests: fix broken "&&" chain
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove a stray "xb" I inadvertently introduced in 780aa0a21e0 (tests:
remove last uses of GIT_TEST_GETTEXT_POISON=false, 2021-02-11).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---

A v2 of a stray patch of mine that wasn't picked up during the release window.

Range-diff:
1:  bd8235ead3c ! 1:  0b43e43b949 mktag tests: fix broken "&&" chain
    @@ Commit message
         mktag tests: fix broken "&&" chain
     
         Remove a stray "xb" I inadvertently introduced in 780aa0a21e0 (tests:
    -    remove last uses of GIT_TEST_GETTEXT_POISON=false, 2021-02-11). This
    -    would have been a failed attempt to type "C-x C-b" that snuck into the
    -    code.
    -
    -    The chainlint check did not catch this one, but I don't know where to
    -    start patching the wall-of-sed that is chainlint.sed to fix that.
    +    remove last uses of GIT_TEST_GETTEXT_POISON=false, 2021-02-11).
     
         Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
     

 t/t3800-mktag.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t3800-mktag.sh b/t/t3800-mktag.sh
index 60a666da595..6275c98523f 100755
--- a/t/t3800-mktag.sh
+++ b/t/t3800-mktag.sh
@@ -17,7 +17,7 @@ check_verify_failure () {
 		grep '$2' message &&
 		if test '$3' != '--no-strict'
 		then
-			test_must_fail git mktag --no-strict <tag.sig 2>message.no-strict &&xb
+			test_must_fail git mktag --no-strict <tag.sig 2>message.no-strict &&
 			grep '$2' message.no-strict
 		fi
 	"
-- 
2.31.0.354.gc8cbd507b5a

